packages/app-store/plausible/components/EventTypeAppCardInterface.tsx

```diff
@@ -24,8 +25,18 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
         }
       }}
       switchChecked={enabled}>
+      <TextField
+        name="Plausible URL"
+        defaultValue="https://plausible.io/js/script.js"
+        placeholder="https://plausible.io/js/script.js"
```

Comment:
Added this placeholder so that if someone removes the entire input text, he knows what kind of inputs goes in it.